packages/core/src/Utils/Analytics/index.ts

```diff
@@ -3,7 +3,7 @@ import Cookies from 'js-cookie';
 import { getLanguage } from '@deriv/translations';
 import { LocalStore, getAppId } from '@deriv/shared';
 import { MAX_MOBILE_WIDTH } from '../../Constants';
-import FIREBASE_INIT_DATA from '../../../../api/src/remote_config.json';
+import FIREBASE_INIT_DATA from '@deriv/api/src/remote_config.json';
```

Comment:
why this change was made? if not intentional please revert this. Revert this even if it was by linting if not intentional

Reply:
Done Vinu, it was done by linting. Thanks 👍